nzedb/Binaries.php

```diff
@@ -637,14 +637,20 @@ public function scan($groupMySQL, $first, $last, $type = 'update', $missingParts
 		// If there was an error, try to reconnect.
 		if ($this->_nntp->isError($headers)) {
 
+			if ($this->first == $this->last) {
```

Comment:
I didn't have a problem with the ternary itself, just having it in the sprintf. This if/else is good to be a ternary.